app/routes/register-your-interest/enter-your-crn.js

```diff
@@ -4,12 +4,14 @@ const Joi = require('joi')
 const urlPrefix = require('../../config/index').urlPrefix
 const callChargesUri = require('../../config/index').callChargesUri
 const ruralPaymentsEmail = require('../../config/index').ruralPaymentsEmail
+const CRN_SCHEMA = require('./crn.schema.js')
 
 const ERROR_MESSAGE = {
   enterYourCrnNumber: 'Enter a CRN',
   enterCrnNumberThatHas10Digits: 'Enter a CRN that has 10 digits',
   confirmYourCrnNumber: 'Confirm your CRN',
-  crnNumbersDoNotMatch: 'The CRNs do not match'
+  crnNumbersDoNotMatch: 'The CRNs do not match',
+  crnNumberOutOfRange: 'The CRN is out of range'
```

Comment:
Sorry for going around the houses on this, lets quickly confirm with Mobeen on the wording here.

Might need to keep it generic and say "The CRN is not valid" or similar. I think this might be bad UX as the customer does not know what range they are meant to adhere to